src/components/ClickableCard/ClickableCard.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getIcon } from '../../utils';
+import Icon from '../shared/Icon';
+import './ClickableCard.scss';
+
+const ClickableCard = ({ children, link=null, handleCardClick=null, iconName=null }) => {
+  // Since either a link or onClick function is required, show an error to the developer if neither are passed in.
+  if (!link && !handleCardClick) {
+    console.error('No click action passed to ClickableCard. Please add a valid `link` or `handleCardClick` prop.');
```

Comment:
I don't think we want to show this error in the browser, even if this situation is true. Could you implement something like what's linked below, such that the PropTypes indicate that one prop or the other are required?

https://stackoverflow.com/a/46337223/2246272

Reply:
As written it actually shows an error in the console, not in the browser- but this looks neat. Have you ever used it? It appears to work, and throws the error in the console when used like this:
```
  link: function(props) {
    if (props['link'] === undefined && props['handleCardClick'] === undefined) {
      return new Error('Please provide either a link or a handleCardClick function.');
    }
    return PropTypes.string;
  },
```

**But** I get the following error:

```checkPropTypes.js:20 Warning: ClickableCard: type specification of prop `link` is invalid; the type checker function must return `null` or an `Error` but returned a function. You may have forgotten to pass an argument to the type checker creator (arrayOf, instanceOf, objectOf, oneOf, oneOfType, and shape all require an argument).```

So despite documentation indicating that this should be possible[1], React is barfing over it. Still investigating but I wanted to see if you had any experience here.

[1] https://reactjs.org/docs/typechecking-with-proptypes.html Search for "custom validation"

Reply:
I'm curious how this error only appears in the console if it's in the render method of the component. Could you try writing this function out as an arrow function?

https://stackoverflow.com/a/49682510/2246272

If it still isn't working, we can leave this conditional in. I'm just surprised it's not working as prescribed in the React docs.

Reply:
Ok, I figured out how to make it work without linter errors or (extra?) errors. The feature seems a teeny bit brittle but I pounded on this implementation a bit and it all seems to behave.